server/datastore/psql/maintenance_requests.go

```diff
@@ -0,0 +1,51 @@
+package psql
+
+import (
+	"time"
+
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+// Import the necessary packages
+type rawTime []byte
+
+func (t rawTime) Time() (time.Time, error) {
+	return time.Parse("15:04:05", string(t))
+}
+
+func (db *DB) AllMaintenanceRequests() []datastore.MaintenanceRequest {
+
+	rows, err := db.connection.Query("select * from public.nsl_maintenance LIMIT $1", 777) //Number limits # of requests, can be anything
+	if err != nil {
+		// handle this error better than this
+		panic(err)
+	}
+
+	var D_Store []datastore.MaintenanceRequest
+	defer rows.Close()
+	for rows.Next() {
+
+		// Create a new MaintenanceRequest
+		var request2 datastore.MaintenanceRequest
+
+		// Scan the row values into MaintenanceRequest
+		err = rows.Scan(&request2.ID, &request2.Request, &request2.UserSubmitted, &request2.DateSubmitted)
```

Comment:
@mikelu92, should we do some string formatting? Right now we return something like: `2020-06-21T23:41:31.332089Z`. Or do we just let frontend handle that?

Reply:
yeah that's fine

Reply:
For clarification, we can just leave as is.